objects/obj_slider_switch_controls/Other_10.gml

```diff
@@ -1,15 +1,15 @@
 /// @description Set controls
 if (instance_exists(obj_player)) {
-	obj_player.use_mouse = custom_value
-	global.use_mouse_for_slider = !custom_value
+	obj_player.use_mouse = custom_value;
+	global.use_mouse_for_slider = !custom_value;
 }
 
 if (custom_value) {
-	window_set_cursor(spr_nothing)
-	cursor_sprite = spr_nothing
-	global.slider_index = 0
-	//global.slider_index = slider_index
+	window_set_cursor(spr_nothing);
+	cursor_sprite = spr_nothing;
+	global.slider_index = 0;
+	//global.slider_index = slider_index;
 }else {
-	window_set_cursor(spr_cursor)
-	cursor_sprite = spr_cursor
+	//window_set_cursor(spr_cursor);
```

Comment:
Do we still need this else statement and commented-out code? if so can you leave a comment as to why its still there?